### PR TITLE
[*] Logout callback wallet deauthentication fix.

### PR DESCRIPTION
**Pull request by MiroMargineanu**

### CODE REVIEW

File: src/utils/logout.ts [hunk 1]
```diff
@@ -9,17 +9,19 @@ export async function logout(
...
     return;
   }
+
   store.dispatch(logoutAction());
+
   try {
     await provider.logout({ callbackUrl });
+
     if (callbackUrl) {
       if (onRedirect) {
```
Comment (IagaruEustatiu) [inline src/utils/logout.ts:23]: can we merge those two ifs in one ?

**Review by IagaruEustatiu — CHANGES_REQUESTED** [review]

File: src/utils/logout.ts [hunk 2]
```diff
@@ -9,17 +9,19 @@ export async function logout(
...
         onRedirect(callbackUrl);
-      } else {
```
Comment (StanislavSava) [inline src/utils/logout.ts:21]: We need to ignore this logic only for the wallet, we need it for other providers

**Review by StanislavSava — CHANGES_REQUESTED** [review]

File: src/utils/logout.ts (hunk 1 above)
Comment (MiroMargineanu) [reply src/utils/logout.ts:24]: No longer necessary.

File: src/utils/logout.ts (hunk 2 above)
Comment (MiroMargineanu) [reply src/utils/logout.ts:21]: Done.

File: src/utils/logout.ts [hunk 3]
```diff
@@ -2,23 +2,28 @@ import { logoutAction } from 'redux/commonActions';
...
+      } else if (getProviderType(provider) !== 'wallet') {
```
Comment (StanislavSava) [inline src/utils/logout.ts:26]: don't use strings, we have a function getIsProviderEqualTo(LoginMethodsEnum.wallet)

**Review by StanislavSava — APPROVED** [review]

**Review by IagaruEustatiu — APPROVED** [review]

**Pull request merged**